challenges/2021_cloud_base_height/03_pytorch_train.ipynb

```diff
@@ -0,0 +1,1039 @@
+{
```

Comment:
<p>it might be worth saying what autologging is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>